types/modules/currency-db.d.ts

```diff
@@ -0,0 +1,60 @@
+import { FirebotUser } from "./user-db";
+
+type Currency = {
+    id: string;
+    name: string;
+    active: boolean;
+    limit: number;
+    transfer: "Allow" | "Disallow";
+    interval: number;
+    payout: number;
+    /** Offline payout */
+    offline: number;
+    bonus: Record<string, number>;
+  };
+
+export type CurrencyDB = {
+    adjustCurrencyForUser: (
+        username: string, 
+        currencyId: string,
+        value: number,
+        adjustType?: string
```

Comment:
I think it would be nice to have a separate `type CurrencyAdjustType = "set" | "adjust";` and use that instead of `string` here and in the other places where it is used.